starter_code/elevator.js

```diff
@@ -1,19 +1,71 @@
+
 class Elevator {
   constructor(){
     this.floor      = 0;
     this.MAXFLOOR   = 10;
     this.requests   = [];
+    this.direction = "up";
+    this.clear = '';
+    this.waitingList = [];
+    this.passengers = [];
+  }
+
+  start() {
+    this.clear = setInterval(() => {
+      this.update()
+    },1000);
+  }
+  stop() {
+    clearInterval(this.clear);
+  }
+  update() {
+    this.log();
   }
+  _passengersEnter() {
+    for(let person in this.waitingList){
```

Comment:
Muy buen uso de for(){} usando let dentro, así person se reduce a su scope local!